src/pybind/mgr/tox.ini

```diff
@@ -152,3 +153,10 @@ modules =
 commands =
     flake8 --config=tox.ini {posargs} \
       {posargs:{[testenv:flake8]modules}}
+
+[testenv:jinjalint]
```

Comment:
this is not integrated into make check: 

https://github.com/ceph/ceph/blob/669660e70602c8fe1d4b2a5887f25993bb26f5f1/src/pybind/mgr/CMakeLists.txt#L9